adb_graphics/figures/maps.py

```diff
@@ -503,6 +502,16 @@ def _title(self):
         if contoured_units:
             contoured = f"{contoured} ({', '.join(contoured_units)}, contoured)"
 
+        return contoured
+
+    def _title(self):
```

Comment:
Title is now implemented differently in the base class and subclass for multipanel plots.